legal.md

```diff
@@ -0,0 +1,99 @@
+
+---
+
+## 4) Commissions and Departments for workplace fairness
+- [US Commission on Civil Rights](https://www.usccr.gov/)
+- [US Department of Justice Civil Rights Division](justice.gov/crt)
+  - [Disability Rights](https://www.justice.gov/crt/disability-rights-section)
+  - [Employment Litigation Section](https://www.justice.gov/crt/employment-litigation-section)
+  - [Office of Special Counsel for Immigration-Related Unfair Employment Practices](https://www.justice.gov/crt/immigrant-and-employee-rights-section)
+- [US Equal Opportunity Employment Commission](https://www.eeoc.gov/)
+
+---
+# Important Laws Surrounding Dismissal
+
+## 1) Layoffs
+
+### [The WARN Act](https://www.dol.gov/sites/dolgov/files/ETA/Layoff/pdfs/WorkerWARN2003.pdf)
+The WARN Act says that employees must receive a **written notice 60 days** before a mass layoff. There are many execptions and conditions to this law that make it difficult to understand and interpret.
+
+- Only companies with **100** or more total US employees must follow this requirement
+- Those employees must be **full time workers** with more than **6 months** tenure at this company to count as a company with "100 or more employees". Part-time employees or newer employees may be eligible for shorter notice periods.
+- A "mass layoff" has a somewhat challenging definition: 
+  - Typically includes 50 or more US workers laid off within a 30 day period
+  - Those 50 workers must be coming from a 'single site'
+    - This gets tricky with remote employment -- if your job is remote, your 'single site' may be:
+      - The location to which you're assigned as a 'home base'
+      - The location from which you're assigned duties
+      - The location to which you report
+- Many employers prefer to pay 60-days severance upon termination as an alternative to delivering a written WARN notice. This technically is not compliant with the WARN act, but *is* compliant with the penalty for violating the WARN act (which is 60 days' backpay and all benefits).
+- Many states have ['mini-WARN' acts](https://edd.ca.gov/en/Jobs_and_Training/Layoff_Services_WARN) which are more stringent or strict in nature -- they may require smaller numbers of people impacted to trigger a WARN notice, for example.
+- There are a number of 'loopholes' that may allow an employer to knowingly violate the WARN act: they may have been seeking funding and giving notice would jeopardize that funding, they may have unreasonable business conditions that prevent them from giving notice, or a natural disaster. 
+- The US Department of Labor or state departments of labor do _not_ enforce WARN. Workers and their representatives must pursue the WARN act through _private right of action_, which is essentially a lawsuit within US court systems with private attorneys. If workers win their case, judges will often assign any legal fees to the former employer.
```

Comment:
This might be a helpful resource to link: https://www.dol.gov/agencies/eta/layoffs/warn